src/main/ui/app/component/NamedList.jsx

```diff
@@ -27,16 +27,13 @@ export default class NamedList extends Component {
         const COLUMNS_IF_SIZE = 5;
```

Comment:
Spare me if I still have missed something. Haven't really had much time due to the exams 😉